packages/cli/src/utils/directory.test.ts

```diff
@@ -8,77 +8,210 @@ const pathsToMatch = (expected: string, desired: string) => {
   return expectedResolved === desiredResolved
 }
 
-describe('directory with base path', () => {
+describe('withBasePath as the current dir `.`', () => {
+  const basePath = '.'
+
   describe('coreDir', () => {
-    it('is the fastsstoreDir + core', () => {
-      expect(pathsToMatch(coreDir, './node_modules/@faststore/core')).toBe(true)
+    it('is the faststoreDir + core', () => {
+      const { coreDir: basedCoreDir } = withBasePath(basePath)
+
+      expect(pathsToMatch(basedCoreDir, './node_modules/@faststore/core')).toBe(true)
+    })
+
+    describe('when is in a monorepo', () => {
+      it.todo('can look at its parent until it reaches the monorepo directory')
     })
   })
 
   describe('tmpDir', () => {
-    it('is the cwd + .faststore', () => {
-      expect(pathsToMatch(tmpDir, './.faststore')).toBe(true)
+    it('is the basePath + .faststore', () => {
+      const { tmpDir: basedTmpDir } = withBasePath(basePath)
+
+      expect(pathsToMatch(basedTmpDir, './.faststore')).toBe(true)
```

Comment:
suggestion (nit): just to match the others

```suggestion
      const { tmpDir: tmpDirWithBase } = withBasePath(basePath)

      expect(pathsToMatch(tmpDirWithBase, './.faststore')).toBe(true)
```